src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowWorkerBuilding.java

```diff
@@ -148,7 +162,14 @@ private void changeDPState()
 
     private void recipeListClicked()
     {
-        @NotNull final WindowListRecipes window = new WindowListRecipes(building.getColony(), building.getPosition());
+        if (!isRecipeAlterationAllowed() && building.getRecipes().isEmpty()) {
```

Comment:
bracket format is still an issue

Reply:
oh my world I've used the other style for years, this is killing me.
Fixed in c49066c.